config/routes.rb

```diff
@@ -147,6 +157,7 @@
   end
 
   get '/admin' => redirect('/admin/conferences')
-
+  conf = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc).first
```

Comment:
This creates as this conference would be passed to the initialize method of the RootRouteConstraint and there `@current.count` becomes undefined. so I am replacing it with 
`conference = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)`